node/src/reactor.rs

```diff
@@ -60,6 +61,9 @@ use crate::{
 use quanta::Clock;
 pub use queue_kind::QueueKind;
 
+/// Lower threshold for percentage of total RAM allocated before dumping queues to disk.
+const MEM_DUMP_THRESHOLD: f32 = 75.0;
```

Comment:
An absolute, configurable value would be better. When running on a cluster, to the best of my knowledge, total memory might appear a lot larger than the allocated/limited memory, e.g. given an allocation and hard limit of 500M for a node, on a machine that has 16G of memory, it should show 16G of total mem even though the process will be killed at 500.1M of memory usage.

Reply:
Fair point, perhaps this should be an absolute value, but optional? @goral09

Reply:
Maybe, I think that absolute value will be a pain to set for node operators or debugging. BUT if we can't have reliable readings with percentage then so it be. It has to be correct first and foremost.

Reply:
Well another issue with percentage is that the total is -system- total, so there may be other processes that add to memory pressure on a single box... leading to the OOM killer stepping in before we actually hit the percentage limit.

Reply:
I thought that using different allocator will help us measure our own usage better and the total is the same regardless of the load on the machine.

Reply:
That's correct, my point was just that other processes may also be consuming memory, so acting on a ratio of `allocated/total` would ignore those other processes, and potentially you could OOM without dumping.

Reply:
Ah, right. Yes I was afraid of that as well. My "solution" was that we should run 1 binary per machine and not put more load on it that could skew the results.